metrics/processors/pod_based_enricher.go

```diff
@@ -182,6 +183,31 @@ func updateContainerResourcesAndLimits(metricSet *core.MetricSet, container kube
 		metricSet.MetricValues[core.MetricMemoryRequest.Name] = intValue(0)
 	}
 
+	for key, val := range container.Resources.Requests {
+		if key == kube_api.ResourceCPU || key == kube_api.ResourceMemory {
```

Comment:
@jingxu97 once you add the metric for local storage, I think you want to add it here as well.

Reply:
Note that we have kube_api.ResourceEphemeralStorage.